src/lib/cache.ts

```diff
@@ -127,7 +127,12 @@ function _get<T>(key) {
           })
         }
       } else {
-        reject(new Error("[Cache#get] Cache miss"))
```

Comment:
I honestly don't _quite_ know why this shows up in datadog, and so this was a bit speculative. These are generally bubbled up to a `.catch` which of course keeps the app running smoothly in terms of responding to a query that caused this error.

However, I _think_ Datadog, when it's sampling for various traces/spans - might _automatically_ pick this up and include it. So this is a bit of a blind fix in the sense of - without fully explaining it, my thinking is we shouldn't have the line: `new Error("Cache miss")` in our codebase, since it's not an error.

There's even some custom tracing code that suppresses the stack trace for these - https://github.com/artsy/metaphysics/blob/adeeaf281a203b2a3644ff9b7218fcef00723b2b/src/lib/tracer.ts#L63